encoder.go

```diff
@@ -25,8 +25,8 @@ func ColoredLevelEncoder(level zapcore.Level, enc zapcore.PrimitiveArrayEncoder)
 
 // ColoredNameEncoder colorizes service names.
 func ColoredNameEncoder(s string, enc zapcore.PrimitiveArrayEncoder) {
-	if len(s) < 12 {
-		s += strings.Repeat(" ", 12-len(s))
+	if len(s) < 20 {
+		s += strings.Repeat(" ", 20-len(s))
 	}
```

Comment:
Because the service plugin displays the plugin name with the specific service name, the name is too long, and 12 characters are obviously not enough

Reply:
Not enough for what?

Reply:
such as :

```yml
service
  sync-data:
    command: xxx
```

the length of 'service.sync-data' is 17

Reply:
I think you don't understand what that code is supposed to do. If the len is, lets say: 17. In that case, the **full** service name would be written without omitting **any** character.
But, if the service name is lets say: 5, 12-5=7 extra spaces would be added to have some extra padding for the small service names.

Reply:
Thus, I'm not really understand your motivation to increase that limit from 12 to 20.

Reply:
I'm sorry. I didn't look closely enough. I made a mistake